src/hw-4_5/routers/login.router.ts

```diff
@@ -0,0 +1,23 @@
+import * as core from 'express-serve-static-core';
+import * as express from 'express';
+import * as jwt from 'jsonwebtoken';
+import { logControllerError } from '../services/errorHandler.service';
+import { jwtSecret } from './routers.consts';
+
+const router: core.Router = express.Router();
+
+router.route('/')
+    .post(async function (req: express.Request, res: express.Response, next: express.NextFunction) {
+        try {
+            const { login, password }  = req.body;
+            const payload = { sub: login, title: password };
```

Comment:
it's not what you should do. In real world example you would have to hash a password and write hashed version to db, then each time when client would like to log in you would hash received password and make a hash to hash check. Don't use payload in a such way, better to write here something you'd use within the session. For instance roles/scopes this user has + maybe some extra information which you can receive anyway from db, but it's not private and can be kept within the payload for faster access.